From ff5e7657d5cbd6916ff05e753215f21482492b51 Mon Sep 17 00:00:00 2001
From: Kalev Lember <klember@redhat.com>
Date: Thu, 6 Dec 2018 09:16:41 +0100
Subject: [PATCH] Fixes to the appdata file

Rename the ID from org.gnucash.Gnucash to org.gnucash.GnuCash to match
flathub, and add a launchable ID to link it to the desktop file.
---
 gnucash/gnome/gnucash.appdata.xml.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/gnucash/gnome/gnucash.appdata.xml.in b/gnucash/gnome/gnucash.appdata.xml.in
index d8f0665811..5318b7774d 100644
--- a/gnucash/gnome/gnucash.appdata.xml.in
+++ b/gnucash/gnome/gnucash.appdata.xml.in
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component>
-  <id type="desktop">org.gnucash.Gnucash</id>
+  <id type="desktop">org.gnucash.GnuCash</id>
+  <launchable type="desktop-id">gnucash.desktop</launchable>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <name>GnuCash</name>
